src/Microsoft.DotNet.Build.Tasks/PackageFiles/FrameworkTargeting.targets

```diff
@@ -106,8 +106,13 @@
   <Import Project="IL.targets"
            Condition="'$(MSBuildProjectExtension)' == '.ilproj' AND '$(SkipImportILTargets)'!='true'" />
 
-  <Import Project="$(MSBuildExtensionsPath32)\Microsoft\Portable\$(TargetFrameworkVersion)\Microsoft.Portable.CSharp.targets"
-          Condition="'$(TargetFrameworkIdentifier)' == '.NETPortable' and '$(MSBuildProjectExtension)' == '.csproj'" />
+   <PropertyGroup Condition="'$(RunningOnCore)' == 'true'">
+     <CSharpCoreTargetsPath>$(MSBuildExtensionsPath32)\Roslyn\Microsoft.CSharp.Core.targets</CSharpCoreTargetsPath>
```

Comment:
I assume these properties are used somewhere in the common targets?

Reply:
Should they instead go in the roslyn targets/props file?

Reply:
These properties will be used by the SDK targets since they will import these files. Build.Common.props or Roslyn.Common.props are both too early since if we set this value there, it will get overriden by the props file that are contained on the roslyn package. One place where we can place it other than here, is on Build.Common.targets, do you think that would be a better place for them?

Reply:
I think this is better than the build.common.targets file.

Reply:
Actually looks like we were only setting the targetsfile here because we wanted to override the CoreCompile task to one that accepted the dotnet tool host when we add the /m to the non-Windows build, but this is not required anymore, I tried removing these lines entirely and build seems to be working by using the CSharp and Vb targets that come with the roslyn package.